### ISSUE TITLE
WCAG2AA issues to fix

### ISSUE THREAD

**Issue by exiguus** [exiguus/zola-monoplain#84]
On the latest a11y audit (https://exiguus.github.io/a11y-audit/zola-monoplain/index.html) some WCAG2AA issues where found:

1. WCAG2AA Principle4 Guideline4_1 4_1_2 H91 InputCheckbox Name
2. WCAG2AA Principle1 Guideline1_4 1_4_3 G18 Fail
3. WCAG2AA Principle1 Guideline1_3 1_3_1 F68

See: https://exiguus.github.io/a11y-audit/zola-monoplain/issues-by-code.html#summary for details
